build.gradle

```diff
@@ -6,15 +6,14 @@ buildscript {
       'compileSdk': 28,
 
       'androidTools': '26.2.0',
-      'kotlin': '1.2.71',
-
+      'kotlin': '1.3.11',
```

Comment:
this version of kotlin is not the latest, but it is recommended in AS 3.3.0 that goes with AGP 3.3.0